stable_baselines3/common/base_class.py

```diff
@@ -420,6 +420,7 @@ def _setup_learn(
             # Make sure training timesteps are ahead of the internal counter
             total_timesteps += self.num_timesteps
         self._total_timesteps = total_timesteps
+        self._num_timesteps_at_start = self.num_timesteps
```

Comment:
Please also add `self._num_timesteps_at_start = 0` in the constructor

Reply:
Sure. Would `None` be more appropriate as a value in the constructor?

Reply:
Ah, okay. `_total_timesteps` is already set to `0` in constructor. Will follow the same convention.